### DISCUSSION TITLE
Getting Application Error for few seconds when move to next page even in production

### DISCUSSION THREAD

**Discussion by Megha-Gupta-Avisoft** [vercel/next.js#67796] [App Router]
Getting Application Error for few seconds when move to next page even in production

**Comment by icyJoseph** [comment]
Check the console, there are useful logs there. Sometimes.

**Reply by Megha-Gupta-Avisoft** [reply]
It appears for few seconds while page is loading , afterward it becomes fine

**Reply by Megha-Gupta-Avisoft** [reply]
> Uncaught (in promise) TypeError: NetworkError when attempting to fetch resource.

 This is console error. But it make page correct after sometime. I am getting application error moving forward and backward and in local it also appears till time api's are loading

**Reply by icyJoseph** [reply]
From the Network tab, can you tell which request/resource is failing to fetch?

**Comment by Megha-Gupta-Avisoft** [comment]
Loading chunk 718 failed.
(error: https://yaptestank.yapsody.com/_next/static/chunks/app/event/index/%5BeventId%5D/layout-5f0a154dc2794850.js)
    NextJS 25   this error also come
Uncaught Error: Minified React error #423; visit https://react.dev/errors/423 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.

**Reply by Megha-Gupta-Avisoft** [reply]
http://testmuskan.staging.yapsody.com/?_rsc=r3yhw  this is the api call

**Reply by icyJoseph** [reply]
Looks like your server/CDN is serving the same content for both the html request, and the rsc request (being loose with the terms here). Where are you hosting?

**Comment by megha-tnt** [comment]
It is hosted in aws

**Reply by icyJoseph** [reply]
Ok, then you need to make sure that cloudfront is not removing the query:

- https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/QueryStringParameters.html

> For distributions, you can choose if you want CloudFront to forward query strings to your origin and whether to cache your content based on all parameters or on selected parameters.

**Comment by Megha-Gupta-Avisoft** [comment]
Can you explain it more to me?

**Reply by icyJoseph** [reply]
What headers do you get on that request (with _rsc) that fails. Post them back here.

**Comment by Megha-Gupta-Avisoft** [comment]
Response Headers: HTTP/1.1 200 OK
venue-code: yaptestank
x-current-path: /
Vary: RSC, Next-Router-State-Tree, Next-Router-Prefetch, Accept-Encoding
X-Powered-By: Next.js
Cache-Control: private, no-cache, no-store, max-age=0, must-revalidate
Content-Type: text/x-component
Content-Encoding: gzip
Date: Wed, 17 Jul 2024 04:02:28 GMT
Connection: keep-alive
Keep-Alive: timeout=5
Transfer-Encoding: chunked

Request Headers: GET /?_rsc=r3yhw HTTP/1.1
Host: yaptestank.yapsody.com
User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:128.0)
Gecko/20100101 Firefox/128.0
Accept: */*
Accept-Language: en-US,en;q=0.5
Accept-Encoding: gzip, deflate, br, zstd
Referer: https://yaptestank.yapsody.com/
RSC: 1
Next-Router-State-Tree:
%5B%22%22%2C%7B%22children%22%3A%5B%22__PAGE__%22%2C%7B%7D%2C%22%2F%22%2C%22refresh%22%5D%7D%2Cnull%2C%22refetch%22%5D
Connection: keep-alive
Cookie: _ga_5M4TQ39BZY=GS1.1.1721188830.98.1.1721188947.5.0.0;
_ga=GA1.1.10395; _fbp=fb.1.1712569586991.1606448153;
_hjSessionUser_67278=eyJpZCI6IjNmMjU0YzlmLThiNDctNTAzNC05YWMxLTA0MzhhYThjMDU4MyIsImNyZWF0ZWQiOjE3MTI1NzAxODAxMjgsImV4aXN0aW5nIjp0cnVlfQ==;
_ga_H04JLE6G3Q=GS1.1.1720185103.27.0.1720185103.60.0.0;
_hp2_id.881785921=%7B%22userId%22%3A%224761479619921613%22%2C%22pageviewId%22%3A%228045410088204633%22%2C%22sessionId%22%3A%228534384672609310%22%2C%22identity%22%3A%22megha.gupta%
40avisoft.io%22%2C%22trackerVersion%22%3A%224.0%22%2C%22identityField%22%3Anull%2C%22isIdentified%22%3A1%7D;
_clck=1sfmi3o%7C2%7Cfni%7C0%7C1568; yap_language=en;
_ga_59XR1MC48F=GS1.2.1720011215.4.1.1720011245.0.0.0;
_ga_HTQS15J1VM=GS1.1.1718974746.6.1.1718974885.60.0.0;
_ga_BJ35QZ9BEK=GS1.1.1718974746.6.1.1718974885.0.0.0;
_ga_0F0V8MZ1LH=GS1.1.1718974746.6.1.1718974885.60.0.0;
_ga_CNG2LFN6XX=GS1.1.1718974746.6.1.1718974885.0.0.0;
_ga_YP9MT2XXFN=GS1.1.1718974747.6.1.1718974885.0.0.0;
_vwo_uuid_v2=DC8F7072522030E24E7E11451A56268D7|65ea1e3f66d64b9e742c203b7683ae18;
_vwo_ds=3%241720067559%3A42.55600232%3A%3A; _vis_opt_s=4%7C;
_gcl_au=1.1.2124073873.1720415212;
_vwo_uuid=DD3F871C9505B8547BC12C3BDD9F855C7; __zlcmid=1MgmrEZqbXu4Ify;
auth_token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzUxMiJ9.eyJpc3MiOiJodHRwOlwvXC95YXBzdXJ2ZXkueWFwc29keS5jb20iLCJhdWQiOiJjaV9pbnRlcmZhY2UiLCJpYXQiOjEzNTY5OTk1MjQsIm5iZiI6MTM1NzAwMDAwMCwiZGF0YSI6eyJpZCI6IjMzMTUiLCJsb2dpbl90aW1lIjoxNzIwNzY4NjAwLCJ0b2tlbl9leHBpcnkiOjE3MjA3Njg2MDB9fQ.KH-YlbIq_j0ajAomQu5APjufgQlxRTSRkzT5RNN46IxTEaL5qqhdswkYbHdKHnd3bKpDCDGs3HqzigKtExDE4w;
sess_auth=7b28c8a0a6f6dce8fd1e99a942846bf9;
yapsodycom-staging=ea257164ccaeb5c229b509448e50410926f7bd50;
api.yapsody.com-production=eyJhbGciOiJIUzI1NiJ9.OTE4MjkwYzAtNDM1Yy0xMWVmLTg5YjctYjM4NDBhNDBiMTU4.NoIutRtAFYX-zzGf1lnTo8GDgcbNYDuE8b1zdrz47Ps.AUyGay5mLEpTx8FEjI%2BMBlH6v25imzhSKeVBHu0RPRA;
_gid=GA1.2.613337270.1721028657;
yapsodycom-production=4d824339090668ba79e35462dd35d19eb72c477a;
_clsk=zz7f5r%7C1721130889443%7C1%7C1%7Cy.clarity.ms%2Fcollect;
_gat_UA-35885324-1=1
Sec-Fetch-Dest: empty
Sec-Fetch-Mode: cors
Sec-Fetch-Site: same-origin
Priority: u=4
Pragma: no-cache

**Comment by Megha-Gupta-Avisoft** [comment]
Also I confirmed from Devopp ,they are not using cloudfront and mostly this
issue replicate in firefox
